RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# Holochain Release process:
+
+## Release candidate test
+
+- Implement fixes, features, changes, and create a release candidate version in the format x.x.x-rc.x
+- Compile a version of the ziptest application in the standalone kangaroo pouch single app runtime using this x.x.x-rc.x version
+- Perform smoke test:
+  - Add images under 50 KiB
+  - Entries must be received by other peers
+    - 0-3 second average reception time: timely (pass)
+    - 3-10 second average reception time: delayed (warn)
```

Comment:
What does "warn" do? Who do we warn? I propose to remove this and extend `pass` to 0-10 seconds.